styles.css

```diff
@@ -722,6 +719,10 @@ textarea {
     justify-content: space-between;
   }
 
+  .none {
+    display: none;
+  }
+
```

Comment:
Very nice work you have done to style the pop-up :clap:

You are almost there, but you still need to style the bottom of the pop-up for the mobile version.

- [ ] In this case, please style the technologies and the buttons to match completely the Figma design

- [ ] Please don't forget to style the buttons accordingly whit the user interactions as active, hover, pressed etc.

**Your design:**
![image](https://user-images.githubusercontent.com/83596135/180838272-b5cf0901-f435-4453-a3d7-07fa5e733e60.png)

**Figma design:**
 
![image](https://user-images.githubusercontent.com/83596135/180838332-5afba682-2c3d-47d0-8118-3fed2ff681e6.png)